pmm-app/src/pmm-qan/panel/components/Details/Table/TableContainer.hooks.ts

```diff
@@ -15,8 +15,7 @@ export const useTables = (example, explains, databaseType): any[] => {
         const parsedJSON = JSON.parse(jsonExplain.value);
 
         const tablesResult = [
-          get(parsedJSON, 'query_block.table.table_name')
-            || get(parsedJSON, 'query_block.ordering_operation.grouping_operation.table.table_name'),
+          get(parsedJSON, 'real_table_name'),
```

Comment:
Meaningless name - mixed const tableResult with `real_table_name`.

Reply:
The names were changed.

Reply:
I would like to call `origin_table_name`, because real sounds strange like for me.

But it's totally optional, you can leave it like this.